packages/js/product-editor/src/blocks/schedule-sale/edit.tsx

```diff
@@ -133,6 +146,7 @@ export function Edit( {}: BlockEditProps< ScheduleSalePricingBlockAttributes > )
 				<div className="wp-block-columns">
 					<div className="wp-block-column">
 						<DateTimePickerControl
+							// ref={ dateOnSaleFromGmtRef }
```

Comment:
Same question here.

Reply:
That is because the DateTimePickerControl is not ForwardedRef yet. That is going to be covered in [#38248](https://github.com/woocommerce/woocommerce/issues/38248).